scout/utils/scout_requests.py

```diff
@@ -319,6 +318,26 @@ def fetch_mim_files(api_key, mim2genes=False, mimtitles=False, morbidmap=False,
     return mim_files
 
 
+def fetch_orpha_files(product4=False, product6=False):
+    """Fetch the necessary files
+    https://www.orphadata.com/data/xml/en_product{nr}.xml
+    Returns:
+        orpha_files(dict): A dictionary with selected files from orphadata
+    """
+    LOG.info("Fetching orpha files from orphadata")
+
+    orphadata_en_product4_url = "https://www.orphadata.com/data/xml/en_product4.xml"
+    orphadata_en_product6_url = "https://www.orphadata.com/data/xml/en_product6.xml"
+
+    orpha_files = {}
+    if product4 is True:
```

Comment:
I don't think we would have any situations where we want to only include only ORPHA terms from one of the files, no?